src/core/traits/scalar.rs

```diff
@@ -386,5 +389,5 @@ fn test_scalar_acos() {
 
     // input is clamped to -1.0..1.0
     assert_eq!(2.0_f64.acos_approx(), 0.0);
-    assert_eq!((-2.0_f64).acos_approx(), core::f64::consts::PI);
+    assert!(((-2.0_f64).acos_approx() - core::f64::consts::PI).abs() < f64::EPSILON);
```

Comment:
I just noticed that I have an approx comparison for the f32 implementation but that macro doesn't work for f64 (glam used to be f32 only, some things haven't gained f64 support).

Reply:
Should i make some change?

Reply:
No, just an aside.